### DISCUSSION TITLE
Welcome to Curvenote Discussions!

### DISCUSSION THREAD

**Discussion by rowanc1** [curvenote/curvenote#643] [Announcements]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is an open community we are building together.

**Comment by eugeniokisho** [comment]
As Typst seems to be a promising replacement for LaTex, will Curvenote implent it to render ".typ" files; https://typst.app/ ?
